src/Test/HttpClient/MockHttpClient.php

```diff
@@ -2,21 +2,19 @@
 
 namespace FOS\HttpCache\Test\HttpClient;
 
-use Http\Adapter\Exception\MultiHttpAdapterException;
-use Http\Adapter\HttpAdapter;
-use GuzzleHttp\Psr7\Response;
+use Http\Client\HttpClient;
 use Http\Discovery\MessageFactoryDiscovery;
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
 
 /**
- * HTTP adapter mock
+ * HTTP client mock
  *
  * This mock is most useful in tests. It does not send requests but stores them
  * for later retrieval. Additionally, you can set an exception to test
  * exception handling.
  */
-class MockHttpAdapter implements HttpAdapter
+class MockHttpClient implements HttpClient
```

Comment:
IIRC the plan was having a separate mock-client package for it. It could go into the utils package, but what if we end up having 2 or 3 more testing utility? Should those also go into the utils package or rather into a separate testing-utils package? #toomuchpackages